src/views/inspector/InspectorViewProvider.ts

```diff
@@ -200,16 +230,26 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
       `<vscode-option value="${variable.key}"${variable.key === this.selectedKey ? ' selected' : ''}>${variable.key}</vscode-option>`
     )) || []
 
-    const featureOptions = this.features && Object.values(this.features).sort((a, b) => (a.name || a.key).localeCompare(b.name || b.key)).map((feature) => (
-      `<vscode-option value="${feature._id}"${feature._id === this.selectedKey ? ' selected' : ''}>${feature.key}</vscode-option>`
-    )) || []
+    const featuresGroupedByName = Object.values(this.features).reduce((acc, feature) => {
+      if (!acc[feature.name]) {
+        acc[feature.name] = []
+      }
+      acc[feature.name].push(feature)
+      return acc
+    }, {} as Record<string, Feature[]>)
+    const featureOptions = this.orderedFeatures.map((feature) => {
+      const featureName = featuresGroupedByName[feature.name]?.length > 1
+        ? `${feature.name} (${feature.key})` 
+        : feature.name
+      return `<vscode-option value="${feature._id}"${feature._id === this.selectedKey ? ' selected' : ''}>${featureName}</vscode-option>`
+    }) || []
```

Comment:
since this is getting a bit verbose, we could pull this out into a separate `getOptions` method that considers the type